src/components/inline/InputText/InputText.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
```

Comment:
Since the rest of this codebase chooses to use `.js` as the file extension for JSX files, I think we should probably follow that convention (i.e. `.ts`) for TSX files. I personally prefer .jsx and .tsx for JSX/TSX files, but the Gatsby tutorial seems to encourage just using `.js`, 🤷.

Reply:
ok